meeting_minutes/220407_SARIF_TC_56.md

```diff
@@ -44,56 +63,86 @@
 
 - Scheduled Teleconferences (Thursdays at 08:00 PDT / 15:00 UTC for 1.5 hours)
 
-    ***On the following date, North America and Europewill be on Daylight Savings Time.***
+    ***On the following date, North America and Europe will be on Daylight Savings Time.***
     ```
     April 21
     ```
+- Proposed Teleconferences (Thursdays at 08:00 PDT / 15:00 UTC for 1.5 hours)
+
+    ```
+    May  5
+    May 26
+    ```
+    Paul has a conflict with May 5 but won't block.
+
 - Possible face-to-face meeting when pandemic permits
 
 # 3. Discussion
 
 ## 3.1 Review recruitment effort to complete the technical committee
 
-* ...
+* We have accumulated new members, including WhiteSource today. OASIS has lined up additional discussions with Apiro. No other recruitment discussion.
+
+### 3.1.1 Discuss difference in emphasis between static and dynamic analysis for potential future members (issue arose during recruitment).
+
+* Static analysis format is mostly fixed, we're moving onto end-to-end fixing/reporting here.
+* Dynamic analysis will entail new design work.
+* Metrics is a new common concern
+* The two domains have commonality in viewing but differences in literal bug-fixing.
+* CyberEO/compliance concerns. NIST has received a directive to provide new rules, etc. https://www.cisa.gov/executive-order-improving-nations-cybersecurity
+* Q: Aharon Abadi: the focus s/be eliminating the problem (and potentially prevention).
+* Katrina asks why aren't we pursuing dynamic analysis? David points out that we need to recharter the TC to pursue this. MF points out we're circling the problem, haven't concretely identified our first primary topic. Katrina isn't sure about MicroFocus involvement in the dynamic analysis topic.
```

Comment:
Well, I didn't mean my comments to be so harsh ;)

Reply:
@katrinaoneil you might rephrase now in case that does better convey your question / statement ... I cannot guess.

Reply:
It's ok -- the gist is right. I just didn't want to sound like I'm being impatient :)